packages/web-components/fast-foundation/CEMToMarkdown.mjs

```diff
@@ -69,7 +69,7 @@ for(var i = 0, modulesLength = modules.length; i < modulesLength; i++)
                 }
                 if(dec.type)
                 {
-                    dec.type.text =  replaceJSDOCLinksWithMDLinks(fixTagsInText(dec.type.text.replaceAll(LF, ' ')));
+                    dec.type.text =  replaceJSDOCLinksWithMDLinks(fixTagsInText(removeCommentBlocks(dec.type.text.replaceAll(LF, ' '))));
```

Comment:
The newline removal should happen after processing, to avoid issues with `// single-line comments `
```suggestion
                    dec.type.text =  replaceJSDOCLinksWithMDLinks(fixTagsInText(removeCommentBlocks(dec.type.text))).replaceAll(LF, ' ');
```

Reply:
I renamed the function to cleanUpVariableTypes and moved the line break removal there since the line breaks need to be cleaned up before the extra spaces are removed. Also the fixTagsInText and repalceJSDOCLinksWithMDLinks calls are not necessary since these things can only exist within the comments which we are removing.